main.go

```diff
@@ -42,19 +42,12 @@ var (
 			String()
 
 	// define prometheus counter
-	nodeAddedTotals = prometheus.NewCounterVec(
+	nodeTotals = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
-			Name: "estafette_gke_preemptible_killer_node_added_totals",
-			Help: "Number of added nodes.",
+			Name: "estafette_gke_preemptible_killer_node_totals",
+			Help: "Number of processed nodes.",
 		},
-		[]string{"name"},
-	)
-	nodeDeletedTotals = prometheus.NewCounterVec(
-		prometheus.CounterOpts{
-			Name: "estafette_gke_preemptible_killer_node_deleted_totals",
-			Help: "Number of deleted nodes.",
-		},
-		[]string{"name"},
+		[]string{"name", "status"},
```

Comment:
The name in this timeline serie will lead to a lot of separate series. Is it worth it? Do you want to know how many inspections before it's deleted?